From 76e12feb81d1b1ccbb7c58b16e040ca6c59f7ace Mon Sep 17 00:00:00 2001
From: Ion Agorria <ion@agorria.com>
Date: Sat, 19 Jan 2019 12:10:23 +0100
Subject: [PATCH] Reverse skip_initramfs to use it by default.

This way it's possible to boot with initram by default.
Initramfs is skip in this device to boot android unless
recovery key is pressed.
---
 init/initramfs.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/init/initramfs.c b/init/initramfs.c
index d39079e690fa..41fb5db74138 100644
--- a/init/initramfs.c
+++ b/init/initramfs.c
@@ -621,8 +621,10 @@ static int __init populate_rootfs(void)
 {
 	char *err;
 
-	if (do_skip_initramfs)
+	if (!do_skip_initramfs) {
+		printk(KERN_INFO "Skipping initramfs...\n");
 		return default_rootfs();
+	}
 
 	err = unpack_to_rootfs(__initramfs_start, __initramfs_size);
 	if (err)
-- 
2.19.2

